packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -1,24 +1,37 @@
+import { Notice } from '@cerner/terra-docs';
 import { Badge } from '@cerner/terra-functional-testing/package.json?dev-site-package';
 
 <Badge />
 
 # Terra Functional Testing
 
-The terra-functional-testing library is a utility for developing automation tests. The library extends the [WebdriverIO Framework](https://webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
+The terra-functional-testing library is a utility for developing automation tests. The library extends [WebdriverIO](https://v6.webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
 
-## Getting Started
+## System Requirements
 
-To get started ensure [docker](https://www.docker.com/) has been installed. The terra-functional-testing library uses docker to setup and run tests in a containerized environment.
+- [Node.js](http://nodejs.org/) Install at least v10.24.0. Due to requirements of fibers used within the project install no greater than lts/erbium (v12).
+- [Docker](https://www.docker.com/) Install docker. Docker is used to setup and run tests in a containerized environment.
 
-Install with npm:
+We strongly recommend using [nvm](https://github.com/nvm-sh/nvm) for installing node versions.
+
+## Installation
+
+Install @cerner/terra-functional-testing and @cerner/terra-cli as development dependencies with npm:
 
 ```
-npm install --save-dev @cerner/terra-functional-testing
+npm install --save-dev @cerner/terra-functional-testing @cerner/terra-cli
 ```
 
-## Test Runner
+## Usage
+
+A test runner is provided for local development. The test runner is invoked via the [terra-cli](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-cli/about).
 
-A test runner is provided for local development and should be invoked using the [terra-cli](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-cli/about).
+package.json
+```json
+"scripts": {
+  "test:wdio": "terra wdio"
+}
+```
```

Comment:
You should also drop the `keepAliveSeleniumDockerService` option from here.

Reply:
Dropped:

https://github.com/cerner/terra-toolkit/pull/610/commits/e6ee8b1b58f5506e29afee2ba4159e16b4a082c8